app/libs/structures/analysis_report.js

```diff
@@ -115,7 +115,7 @@ class AnalysisReport {
       include: [
         {model: db.models.patientInformation, as: 'patientInformation', attributes: {exclude: ['id', 'deletedAt']}},
         {model: db.models.user.scope('public'), as: 'createdBy'},
-        {model: db.models.template.scope('public'), as: 'template'},
+        {model: db.models.template.scope('minimal'), as: 'template'},
```

Comment:
I'll have to make some client changes for this as it uses the sections from the report endpoint currently

Reply:
I switched it to minimal because templates contain images and it's not a good idea to store image data in a cache. The template ident is still returned with the report though, so you should be able to use that to query /templates/{ident} to get the template data.

Reply:
Yep that's fine, I'll just have to make a client PR to account for that before merging this